### PR TITLE
[plugin.audio.radiobrowser] 1.2.0

### PR DESCRIPTION
**Pull request by segler-alex**
### Description
This is an update of the plugin audio radiobrowser to version 1.2.0 with the following changes:
* use new api as documented at https://api.radio-browser.info
* stop play before start again to fix play only every second time bug
* use xbmc log instead of print

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.audio.radiobrowser/main.py [hunk 1]
```diff
@@ -196,37 +262,40 @@ def delFromMyStations(stationid):
...
 
 elif mode[0] == 'play':
-    stationid = args['stationid'][0]
-    data = downloadFile('http://www.radio-browser.info/webservice/v2/json/url/'+str(stationid),None)
+    stationuuid = args['stationuuid'][0]
+    data = downloadApiFile('/json/url/' + str(stationuuid),None)
     dataDecoded = json.loads(data)
     uri = dataDecoded['url']
+    xbmc.Player().stop()
     xbmc.Player().play(uri)
```
Comment (enen92) [inline plugin.audio.radiobrowser/main.py:278]: Plugins should not call Player.play() but use xbmcplugin.setResolvedUrl instead. Can you adjust the add-on?

Comment (segler-alex) [reply plugin.audio.radiobrowser/main.py:277]: yes i will immediatelly

Comment (segler-alex) [reply plugin.audio.radiobrowser/main.py:277]: it does not work :(
maybe the wrong place to ask, but do you know of any reason why the addon handle can get the value of "-1" ?
addon_handle = int(sys.argv[1])

Comment (segler-alex) [reply plugin.audio.radiobrowser/main.py:277]: isFolder=True does make the handle transfer work again.... that is why it works for all other pages but the deepest one where the real streams are...

Comment (segler-alex) [reply plugin.audio.radiobrowser/main.py]: i found the solution, i was missing
li.setProperty('IsPlayable', 'true')

Comment (enen92) [reply plugin.audio.radiobrowser/main.py]: Yeah sorry for the delay. You need to set the IsPlayable property to true to make it work. The reason behind this request is that, if you don't do it, everytime you play a radio channel a new entry is added to the database. Plugin setresolvedurl assigns a unique path to each of the items, hence, you're always playing the same file although the resolved stream url might difer

**Pull request merged**